controllers/api.controller.js

```diff
@@ -0,0 +1,8 @@
+const getAllEndpoints = require("../models/api.model")
+
+function getEndpoints (req, res, next) {
+    const allEndpoints = getAllEndpoints()
+    res.status(200).send(allEndpoints)
```

Comment:
To keep with convention, ensure this is sent back on a properly named key on an object